doc/nrf/external_comp/bt_fast_pair/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _ug_bt_fast_pair:
+
+Google Fast Pair integration
+############################
+
+Google Fast Pair is a standard for pairing *Bluetooth®* and Bluetooth Low Energy (LE) devices with as little user interaction required as possible.
+Google also provides additional features built upon the Fast Pair standard.
+For detailed information about supported functionalities, see the official `Fast Pair`_ documentation.
+
+The subpages include additional pages that discuss the Fast Pair helper modules.
```

Comment:
The bt_fast_pair guide is the core (main) guide for the Fast Pair integration. The guide for the advertising manager is the only optional guide if the user wants to use this particular module. The overall Fast Pair integration is possible without this helper guide.

I think you need to change this part.